### DISCUSSION TITLE
Feedback after a fresh first time install

### DISCUSSION THREAD

**Discussion by ScopeSV** [zed-industries/zed#6810] [Feature Requests]
Hi. I just read about zed as it's going open source, and wanted to check it out. 
I just downloaded version 0.119.19 on macOS.

1.
First I tried to open a typescript project.
Whenever I selected a typescript file, it switched between:
- Downloading typescript server
- Checking for updates
Multiple times before it stopped.
It does not seem like the typescript server is running, and its just working as a dumb text editor.

I tried loading up a Go project.
This worked better. Got the same "Downloading go language server" and everything seems to work good.
I do miss the auto import feature you get in both vim and vscode though. 

2.
I cant figure out how to run tests from within the editor with typescript or go.

3.
Are there any git operations available in the editor? I see a collaborate tab with a GitHub login. This is not the same as git though.

4.
I tried clicking on the Inline assist button. It takes me to a window where it says gpt-4-turbo and I can toggle between multiple AIs. I cant figure out how to use this. I tried writing some stuff here. Am I suppose to get some sort of feedback from an AI here? If so, it's not working. 

---------
Zed feels very fast and snappy. Seems very performant. The vim bindings seems to work well enough for me. Not sure if there is a setting for relative line numbers?

But unless I'm using it wrong, it lacks quite a bit before I can use / test it as my daily driver. Thanks for making an effort creating a new editor. We really need something that can take vscode down from its pedestal. Looking forward to testing it once it matures a bit more, but as it doesn't seem to support the typescript language its a bit too much manual work :-)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
